terraform/modules/cloudtrail/cloudtrail-cloudwatch/variables.tf

```diff
@@ -0,0 +1,8 @@
+variable "trail_name" {
+  description = "The name of the CloudTrail trail we are setting up CloudWatch for."
+}
+
+variable "retention_in_days" {
+  default     = 730
```

Comment:
We're using 731 above, maybe this should match.

Reply:
Good call, 730 won't run, it's not a valid choice iirc